README.md

```diff
@@ -41,7 +41,7 @@ from openapi_tester import SchemaTester
 schema_tester = SchemaTester(schema_file_path="./schemas/publishedSpecs.yaml")
 ```
 
-Once you've instantiated a tester, you can use it to test responses:
+Once you've instantiated a tester, you can use it to test responses and requests (on successful responses):
```

Comment:
Don't think it's necessary to explain this detail here :+1: 
```suggestion
Once you've instantiated a tester, you can use it to test responses and request bodies
```

Reply:
Done 👍